src/integrations/admin/social-templates-integration.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+namespace Yoast\WP\SEO\Integrations\Admin;
+
+use WP_Taxonomy;
+use WPSEO_Admin_Editor_Specific_Replace_Vars;
+use WPSEO_Admin_Recommended_Replace_Vars;
+use WPSEO_Replacevar_Editor;
+use WPSEO_Shortlinker;
+use Yoast\WP\SEO\Conditionals\Open_Graph_Conditional;
+use Yoast\WP\SEO\Config\Badge_Group_Names;
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+use Yoast_Form;
+use function esc_html__;
+use function esc_url;
+use function YoastSEO;
```

Comment:
Not sure about these 3 `use function`. Not sure we're importing functions this way anywhere, unless I'm missing something. Any specific reason for this?

Reply:
You're right, probably it was my IDE when I moved the class. Those should definitely go.